gravitee-am-management-api/gravitee-am-management-api-rest/src/main/resources/views/default/registration.html

```diff
@@ -170,6 +170,6 @@
 
     const passwordSettings = /*[[${passwordSettings}]]*/ null;
```

Comment:
Missing ```const passwordValidation = /*[[${passwordValidation}]]*/ null;``` no ?

Reply:
No, this is the variable used to store the GW URL that the v3 script need to call in order to valdiate the password.
As for registration page we rely on the form fields to evaluate the "User Profile Information" rule, we do not need for this value and the GW will not provide this variable during the template rendering